src/test/java/com/aws/greengrass/clientdevices/auth/iot/registry/ThingRegistryTest.java

```diff
@@ -1,115 +0,0 @@
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.Mock;
-import org.mockito.junit.jupiter.MockitoExtension;
-
-import java.io.IOException;
-import java.util.Collections;
-
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.is;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.mockito.Mockito.any;
-import static org.mockito.Mockito.doThrow;
-import static org.mockito.Mockito.reset;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
-
-@ExtendWith({MockitoExtension.class, GGExtension.class})
-class ThingRegistryTest {
-    private static final String mockThingName = "mock-thing";
-    private static final Thing mockThing = Thing.of(mockThingName);
-    private static Certificate mockCertificate;
-
-    @Mock
-    private IotAuthClient mockIotAuthClient;
-    private Topics configTopics;
-    private DomainEvents domainEvents;
-    private ThingRegistry registry;
-
-    @BeforeEach
-    void beforeEach() throws InvalidCertificateException {
-        domainEvents = new DomainEvents();
-        configTopics = Topics.of(new Context(), "config", null);
-        registry = new ThingRegistry(mockIotAuthClient, domainEvents, RuntimeConfiguration.from(configTopics));
-        mockCertificate = CertificateFake.of("mock-certificateId");
-    }
-
-    @AfterEach
-    void afterEach() throws IOException {
-        configTopics.context.close();
-    }
-
-    @Test
-    @SuppressWarnings("PMD.CompareObjectsWithEquals")
-    void GIVEN_emptyRegistry_WHEN_createThing_THEN_thingIsAddedToRegistry() {
-        Thing createdThing = registry.createThing(mockThingName);
-        Thing retrievedThing = registry.getThing(mockThingName);
-
-        assertThat(createdThing.getThingName(), is(mockThingName));
-        assertThat(createdThing.getAttachedCertificateIds(), equalTo(Collections.emptyMap()));
-
-        assertThat(createdThing, equalTo(retrievedThing));
-        // IMPORTANT! Ensure a copy is returned. It is incorrect to call the equals method for this
-        assertThat(createdThing != retrievedThing, is(true));
-
-        // TODO: check ThingUpdated event
-    }
-
-    @Test
-    void GIVEN_unmodifiedThing_WHEN_updateThing_THEN_oldThingReturned() {
-        Thing createdThing = registry.createThing(mockThingName);
-        Thing returnedThing = registry.updateThing(createdThing);
-        assertThat(createdThing, equalTo(returnedThing));
-
-        // TODO: no update event
-    }
-
-    @Test
-    void GIVEN_validThingAndCertificate_WHEN_isThingAttachedToCertificate_THEN_pass() {
```

Comment:
This file just got moved to a different folder. It is the same as the one above. I remove this last 2 tests that where directly testing the `registry.isThingAttachedToCertificate` that doesn't exist anymore. I will cover all this with integration tests for the offline flow